cmd/checkmetrics/ci_worker/checkmetrics-json-qemu-sv-c1-small-x86-01.toml

```diff
@@ -68,7 +68,7 @@ description = "measure container average of blogbench read"
 # within (inclusive)
 checkvar = ".\"blogbench\".Results | .[] | .read.Result"
 checktype = "mean"
-midval = 55011.88
```

Comment:
@GabyCT Do we know what Pr caused this?

Reply:
Please, @amshinde, take a look at https://github.com/kata-containers/tests/issues/5261.

Reply:
@fidencio thanks for the pointer.

Reply:
@GabyCT Would appreciate it if we could mention the golang update causing this, somewhere in the PR commit message.